testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/ee/datasourcedefinition/DataSourceDefinitionTestCase.java

```diff
@@ -120,4 +125,24 @@ public void testEmbeddedDatasource() throws NamingException, SQLException {
         DataSourceBean bean = (DataSourceBean)ctx.lookup("java:module/" + DataSourceBean.class.getSimpleName());
         Assert.assertEquals(bean.getDataSource5().getConnection().nativeSQL("dse"),"dse");
     }
+
+    @Test
+    /**
+     * Test if NullPointerException ir raised when connection is closed and then method connection.isWrapperFor(...) is called
+     * See https://issues.jboss.org/browse/JBJCA-1389
+     */
+    public void testCloseConnectionWrapperFor() throws NamingException, SQLException {
+        expectedException.expect(SQLException.class);
+        expectedException.expectMessage("IJ031041: Connection handle has been closed and is unusable");
```

Comment:
We should probably just check for the message id, or it will fail if someone is e.g. using German as their locale